### ISSUE TITLE
Types incompatible with TS 4.4

### ISSUE THREAD

**Issue by adidahiya** [palantir/redoodle#63]
tested with TS 4.4.3 only, so far. I get compilation errors within redoodle types:

```
[error] TypeScript error node_modules/.pnpm/redoodle@2.6.1/node_modules/redoodle/lib/Store.d.ts:24:5 - error TS1169: A computed property name in an interface must refer to an expression whose type is a literal type or a 'unique symbol' type.

24     [Symbol.observable](): _ObservableLike<S>;
```

**Comment by garylhua** [comment]
bump

**Comment by pgoldberg** [comment]
Fixed in `redoodle@2.9.0`
